integration/simulation/simulation_in_mem_test.go

```diff
@@ -10,10 +10,13 @@ import (
 // Everything else is reported to this value. This number has to be adjusted in conjunction with the number of nodes. If it's too low,
 // the CPU usage will be very high during the simulation which might result in inconclusive results.
 func TestInMemoryMonteCarloSimulation(t *testing.T) {
+	logFile := setupTestLog()
+	defer logFile.Close()
+
 	params := SimParams{
 		NumberOfNodes:         10,
 		NumberOfWallets:       5,
-		AvgBlockDurationUSecs: uint64(30_000),
+		AvgBlockDurationUSecs: uint64(40_000),
```

Comment:
was it failing frequently?

Reply:
Yeah, in CI. Which is weird, as the in-memory stuff is basically untouched. I wondered if it was a result of doing more in parallel (since the socket tests are now uncommented).

Reply:
So fewer hardware resources. Yeah, possibly.